packages/discord.js/src/client/actions/AuditLogEntryCreate.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const Action = require('./Action');
+const GuildAuditLogsEntry = require('../../structures/GuildAuditLogsEntry');
+const Events = require('../../util/Events');
+
+class AuditLogEntryCreateAction extends Action {
+  handle(data) {
+    const client = this.client;
+    const guild = client.guilds.cache.get(data.guild_id);
+    let auditLogEntry;
+
+    if (guild) {
+      auditLogEntry = new GuildAuditLogsEntry(guild, data);
+
+      /**
+       * Emitted whenever a guild audit log entry is created.
+       * @event Client#auditLogEntryCreate
+       * @param {Guild} guild The guild where the entry was created
+       * @param {AuditLogEntry} auditLogEntry The entry that was created
```

Comment:
I believe `AuditLogEntry` is more important and should be emitted as the first parameter. It would also be consistent with other events.